docs/before-you-begin.md

```diff
@@ -1,32 +1,64 @@
-        1. Open a Terminal window
-        2. Run the following command:
-            ```bash
-            kit version
-            ```
-        3. You should see output similar to the following:
-        
-            Version: 0.4.0<br>
-            Commit: e2e83d953823ac35648f2f76602a0cc6e8ead819<br>
-            Built: 2024-11-05T20:29:07Z<br>
-            Go version: go1.22.6<br>
+This project was created using Python v3.12, but works with Python versions >= 3.10.
+
+### 1/ Install the Kit CLI
+
+The PyKitOps SDK uses the Kit CLI to manage ModelKits so you'll need an up-to-date Kit CLI version on your machine.
+
+To determine if the Kit CLI is installed in your environment:
+1. Open a Terminal window
+1. Run the following command:
+
+   ```bash
+   kit version
+   ```
+1. You should see output similar to the following:
+
+    ```
+    Version: 0.4.0
+    Commit: e2e83d953823ac35648f2f76602a0cc6e8ead819
+    Built: 2024-11-05T20:29:07Z
+    Go version: go1.22.6
+    ```
+
+If you don't have the Kit CLI installed, follow the [Kit Installation Instructions](https://kitops.ml/docs/cli/installation.html).
+
+### 2/ Prepare Your Registry
 
+To get the most out of ModelKits we strongly suggest you [sign up for a free account at Jozu.ml](https://api.jozu.ml/signup).
 
-3. If you don't have the Kit CLI installed, follow the [Kit Installation Instructions](https://kitops.ml/docs/cli/installation.html).
-4. If you haven't already done so, [sign up for a free account with Jozu.ml](https://api.jozu.ml/signup) so you can store your ModelKits in its purpose-built registry.
-5. In the root directory of your project--which we'll call the *Project directory*--create a `.env` file.
-6. Edit your `.env` file by adding an entry for your **JOZU_USERNAME**, your **JOZU_PASSWORD** and your **JOZU_NAMESPACE** (aka your **Personal Organization** name). For example:
+The [Jozu Hub](https://jozu.ml/) will:
+* Automatically generates a container from a ModelKit
+* Show you details about the various parts of your ModelKit at a glance
+* Indicate whether ModelKits are signed
 
+Alternatively, ModelKits can be stored in any OCI 1.1-compliant container registry, however, you'll need to set the `JOZU_REGISTRY` environment variable in addition to the username, password, and namespace (see the next section for details).
+
+### 3/ Set Your Environment
+
+1. In the root directory of your project (the *"Project directory"*) create a `.env` file.
+1. Edit the `.env` file by adding an entry for your `JOZU_USERNAME`, your `JOZU_PASSWORD` and your `JOZU_NAMESPACE` (this should match the repository name you'll be pushing to in the regsitry). If you're *not* using the Jozu Hub you'll also need to set the `JOZU_REGISTRY` variable to point to the URL for your registry.
+
+   An example `.env` file for Jozu Hub will look like this:
+   
+        ```
+        JOZU_USERNAME=brett@jozu.org
+        JOZU_PASSWORD=my_password
+        JOZU_NAMESPACE=brett
+        ```
```

Comment:
Double check how GitHub is parsing the format in this file -- in markdown, both indenting by 4+ spaces and fencing a block with ``` are treated as code blocks. Since this is 5 spaces in, it's getting parsed as 

        ```
        JOZU_USERNAME=brett@jozu.org
        JOZU_PASSWORD=my_password
        JOZU_NAMESPACE=brett
        ```

see: https://github.com/jozu-ai/pykitops/blob/55120a8fb11c0a1ce8936da4b71d9152cb4ceb99/docs/before-you-begin.md